frontend/src/lib/components/FrontPageAccordion.svelte

```diff
@@ -2,17 +2,31 @@
 
 <script lang="ts">
 import { i18n } from "$lib/i18n.svelte";
+import DOMPurify from "dompurify";
 import { Accordion, AccordionItem } from "flowbite-svelte";
+
+const sanitizeConfig = {
+	ALLOWED_TAGS: ["strong", "p", "i", "ul", "li", "ol"],
+	ALLOWED_ATTR: [], // explicitly none
+};
+
+function sanitizeHtml(html) {
+	return DOMPurify.sanitize(html, sanitizeConfig);
+}
 </script>
 
-   <Accordion class="min-w-full px-4 mt-10">
-        <AccordionItem >
+<Accordion class="min-w-full px-4 mt-10">
+    <AccordionItem>
         <span slot="header">{i18n.tr.frontpageAccordion.heading1}</span>
-      <p class="mb-4 mt-4 px-4 text-gray-700 dark:text-gray-400">{@html i18n.tr.frontpageAccordion.summary1}</p>
+        <div class="mb-4 mt-4 px-4 text-gray-700 dark:text-gray-400 whitespace-pre-line restore-default">
```

Comment:
If we're rendering html then (at least to me) it's surprising that a newline in the html source is also rendered as a newline - I'd suggest we either do one or the other but not both:
```suggestion
        <div class="mb-4 mt-4 px-4 text-gray-700 dark:text-gray-400 restore-default">
```

Reply:
That's true, if the user puts a newline inbetween the bullet points (which I fixed in my translation values), it appears. You're right I'll get rid of that.